docs/ENVIRONMENT.md

```diff
@@ -471,3 +471,25 @@ SYSTEMD_HOME_DEBUG_SUFFIX=foo \
   in a more compact format that reduces the amount of disk space required by the
   journal. Note that journal files in compact mode are limited to 4G to allow use of
   32-bit offsets. Enabled by default.
+
+* `$SYSTEMD_JOURNAL_COMPRESS` – Takes a boolean, or one of the compression
+  algorithms "XZ", "LZ4", and "ZSTD". If enabled, the default compression
+  algorithm set at compile time will be used when opening a new journal file.
+  If disabled, the journal file compression will be disabled. Note that the
+  compression mode of existing journal files are not changed. To make the
+  specified algorithm takes an effect immediately, you need to explicitly run
+  `journalctl --rotate`.
+
+`systemd-pcrphase`, `systemd-cryptsetup`:
```

Comment:
Please drop this part. The commit only adds `$SYSTEMD_JOURNAL_COMPRESS`.

Reply:
dropped